### PR TITLE
Fix DnD tests for WebDriver BiDi

### PR DESCRIPTION
**Pull request by jrandolf**

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

File: webdriver/tests/bidi/input/perform_actions/pointer_mouse.py [hunk 1]
```diff
@@ -211,44 +211,76 @@ async def test_drag_and_drop(
...
+                                             context=top_context["context"])
 
     # mouseup that ends the drag is at the expected destination
-    events = await get_events(bidi_session, new_context["context"])
+    events = await get_events(bidi_session, top_context["context"])
 
-    assert len(events) >= 5
-    assert events[1]["type"] == "dragstart", f"Events captured were {events}"
-    assert events[2]["type"] == "dragover", f"Events captured were {events}"
+    assert len(events) >= 6
```
Comment (jgraham) [inline webdriver/tests/bidi/input/perform_actions/pointer_mouse.py:240]: We check for five specific events below, do we definitely get 6, even if we don't expect `dragleave`?

File: webdriver/tests/bidi/input/perform_actions/pointer_mouse.py [hunk 2]
```diff
@@ -211,44 +211,75 @@ async def test_drag_and_drop(
...
-        .pointer_up(button=0)
-        .pointer_move(80, 50, duration=100, origin="pointer")
-    )
+    actions.add_pointer().pointer_move(
+        x=0, y=0, origin=get_element_origin(drag_target)).pointer_down(
+            button=0).pointer_move(
+                x=0,
+                y=0,
+                duration=drag_duration,
+                origin=get_element_origin(drop_target)).pointer_up(button=0)
```
Comment (whimboo) [inline webdriver/tests/bidi/input/perform_actions/pointer_mouse.py:232]: With those changes the individual action steps are very hard to read. Would it be fine with you to revert it by using surrounding brackets again and having each action on its own line?

Comment (jrandolf) [reply webdriver/tests/bidi/input/perform_actions/pointer_mouse.py]: Done.

**Comment by whimboo** [comment]
As @jgraham mentioned on Matrix please rebase your branch against latest master so that the tests will run correctly. Thanks.

**Review by jgraham — APPROVED** [review]

**Pull request merged**